README.md

```diff
@@ -105,3 +105,32 @@ The CI builds redistributable release binaries automatically.
 Follow the CI status link from the [commits page](https://github.com/Zubax/kucher/commits/master)
 to find the binaries for a particular commit.
 The Linux binaries can be shipped directly; the Windows binaries must be signed first.
+
+### Signing Windows executables
+
+Requirements:
+* Windows 10
+* An unsigned executable file named `Kucher.exe`
+* Certum cryptographic card (USB dongle), with a valid certificate installed
+* Signtool: available in [Windows development Kit](https://developer.microsoft.com/en-us/windows/downloads/windows-10-sdk)
+
+1 - Connect the USB dongle to the PC. Launch "Start.exe", click "Install Code Signing" and follow the instructions. 
```

Comment:
It's not entirely clear where this file is located. Is it on the dongle?

Reply:
"Start.exe" is located on the card, you can see it when you open the dongle